atlasdb-api/src/main/java/com/palantir/atlasdb/config/LeaderConfig.java

```diff
@@ -98,6 +99,11 @@ public Duration leaderPingResponseWait() {
         return Duration.ofMillis(leaderPingResponseWaitMs());
     }
 
+    @Value.Default
+    public HumanReadableDuration leaderAddressCacheTtl() {
+        return HumanReadableDuration.seconds(15);
```

Comment:
Do we need to have this be configurable?

Reply:
Also, I don't see this actually is used?

Reply:
Perhaps initially, whilst we iron it out. It may not need to go all the way to the config, but for tests it's a bit easier if it's configurable at the lower levels.

Reply:
Yeah, it looks like you missed wiring in Leaders#createInstrumentedLocalServices.

Reply:
I think it's fine for it to be injectable to the class as a param, so in tests you can turn it off. But then there should just be a default in the default constructor that's used by prod code.

Reply:
still need it to be on `LeaderConfig`, I've made it `@JsonIgnore` however, so it's only done through code. I've removed it off of `PaxosRuntimeConfiguration`.

It only needs to be on `LeaderConfig` because timelock creates a leader config and calls `Leaders#createInstrumentedLocalServices` as opposed to using the builder itself. This goes away with my partitioning work, since that path is completely changed. Once I bring that back, I can remove it off of `LeaderConfig`.